### ISSUE TITLE
panic: attach flag redefined: detach-keys

### ISSUE THREAD

**Issue by abitrolly** [containers/libpod#5927]
/kind bug

**Description**

I've compiled `podman` from sources, but when I run it, it fails.
```
⬢[anatoli@toolbox libpod]$ ./bin/podman
panic: attach flag redefined: detach-keys

goroutine 1 [running]:
github.com/spf13/pflag.(*FlagSet).AddFlag(0xc0000f5600, 0xc000341540)
        vendor/github.com/spf13/pflag/flag.go:848 +0x807
github.com/spf13/pflag.(*FlagSet).VarPF(0xc0000f5600, 0x1d1a980, 0x29a7ea0, 0x19f1c5b, 0xb, 0x0, 0x0, 0x1a8f0d9, 0xce, 0x44d76e)
        vendor/github.com/spf13/pflag/flag.go:831 +0x10b
github.com/spf13/pflag.(*FlagSet).VarP(...)
        vendor/github.com/spf13/pflag/flag.go:837
github.com/spf13/pflag.(*FlagSet).StringVar(0xc0000f5600, 0x29a7ea0, 0x19f1c5b, 0xb, 0xc0003eeab0, 0xd, 0x1a8f0d9, 0xce)
        vendor/github.com/spf13/pflag/string.go:37 +0x9b
github.com/containers/libpod/cmd/podman/containers.attachFlags(0xc0000f5600)
        cmd/podman/containers/attach.go:64 +0x84
github.com/containers/libpod/cmd/podman/containers.init.1()
        cmd/podman/containers/attach.go:79 +0x12a
```

**Steps to reproduce the issue:**

1.
```
toolbox enter
```
2.
```
sudo dnf install make golang gpgme-devel libseccomp-devel
```
3.
```
⬢[anatoli@toolbox libpod]$ make podman
Podman is being compiled without the systemd build tag. Install libsystemd on   Ubuntu or systemd-devel on rpm based distro for journald support.
GO111MODULE=on go build -mod=vendor  -gcflags 'all=-trimpath=/home/anatoli/f/libpod' -asmflags 'all=-trimpath=/home/anatoli/f/libpod' -ldflags '-X github.com/containers/libpod/libpod/define.gitCommit=0d817f5210defac51cc86c60ec86bcefca0511c9 -X github.com/containers/libpod/libpod/define.buildInfo=1587495349 -X github.com/containers/libpod/libpod/config._installPrefix=/usr/local -X github.com/containers/libpod/libpod/config._etcDir=/etc -extldflags ""' -tags "ABISupport  exclude_graphdriver_btrfs btrfs_noversion selinux  exclude_graphdriver_devicemapper seccomp varlink" -o bin/podman github.com/containers/libpod/cmd/podman
⬢[anatoli@toolbox libpod]$ echo $?
0
```

**Additional information you deem important (e.g. issue happens only occasionally):**

If it is a missing library, I'd expect the build process to fail. It is Fedora 31.

**Comment by abitrolly** [comment]
Fixed by #5923. @baude How could it escape tests?

**Comment by baude** [comment]
@abitrolly https://podman.io/blogs/2020/04/16/podman-v2-announce.html
